_posts/2020-01-21-200wad-logo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: 200WaD - Logo
+date: 2020-01-21 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Note: This post is about the new logo @ [200WaD][1]. If you're not using the website you might not be interested in it, feel free to skip it.
+
+I really liked the original 200 words a day logo (the old logo) even though baz said multiple times that he made it himself and didn't really take too much time with it. I consider that a plus since it came out really well despite that.
```

Comment:
What's the  boyfriend in () for? Or just repeats stuff... Unless it's a link?

Reply:
Spell people's names with big letters

Reply:
> What's the boyfriend in () for? Or just repeats stuff... Unless it's a link?

Yeah wanted to add it, but could not find it. So will remove it for now